Slack.NetStandard/Messages/Elements/DispatchActionConfig.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Slack.NetStandard.Messages.Elements
+{
+    public class DispatchActionConfig
+    {
+        public DispatchActionConfig(ActionTrigger[] triggers)
```

Comment:
Could we have a parameterless constructor for this please, it will allow the object to be de-serialized without raising errors.